### PR TITLE
Build: Update to QUnit 2

### PR DESCRIPTION
**Pull request by Krinkle**
See <https://qunitjs.com/upgrade-guide-2.x/>. Basically just two trivial renames that landed in QUnit 1.x with compat aliases that were removed in 2.0.

The branch contains two other changes:
1. To fix the commitplease build failure.
2. To pull in ~~an unmerged fix at https://github.com/rxaviers/zoned-date-time/pull/3~~ the updated version of `zoned-date-time` to address the ReferenceError about `module` being undefined.

### CODE REVIEW

**Comment by rxaviers** [comment]
Thank you

File: package.json [hunk 1]
```diff
@@ -95,8 +95,9 @@
     "iana-tz-data": ">=2017.0.0",
     "matchdep": "1.0.1",
     "mocha": "^3.4.2",
+    "qunit": "2.12.0",
     "semver": "^5.3.0",
-    "zoned-date-time": "1.0.0"
+    "zoned-date-time": "https://github.com/Krinkle/zoned-date-time#bbf87839ef7e8b8dfb8ceec94b2f5bad3aaef7b6"
```
Comment (rxaviers) [inline package.json:100]: This seems unrelated to QUnit change. Please, do you mean it?

Comment (Krinkle) [reply package.json:100]: Yes and no. See point 2 in the pull description. It is needed for it to pass. But I do not recommend merging it in this way, instead we should probably land https://github.com/rxaviers/zoned-date-time/pull/3 first and release that, then you or I can update the version number here.

Comment (rxaviers) [reply package.json:100]: Thanks, `zoned-date-time@1.1.0` released.

Comment (Krinkle) [reply package.json]: I've updated the PR.

**Comment by rxaviers** [comment]
Awesome thank you
